test/expects/pug.js

```diff
@@ -1,3 +1,3 @@
-var pug = { template: "<div class=\"pug__test keep-me\"><article><p>foo</p></article></div>",cssModules: {"test":"pug__test"},};
+var pug = { template: "<div class=\"pug__test keep-me\" v-if=\"true\"><article><p>foo</p></article></div><p v-else=\"v-else\">nothing</p>",cssModules: {"test":"pug__test"},};
```

Comment:
Does `v-else` correctly work here? It seems pug cannot know `v-else` is a boolean attribute but Vue compiler does not care. I'm not a pug user personally. Maybe @aleksejevski  can give a comment.